src/main/java/dev/dkvl/womutils/panel/MiscInfo.java

```diff
@@ -3,20 +3,19 @@
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 
-import javax.swing.*;
-
 @Getter
 @AllArgsConstructor
 public enum MiscInfo
 {
-    BUILD("Build", new JLabel("--")),
-    COUNTRY("Country", new JLabel("--")),
-    TTM("TTM", new JLabel("--")),
-    EHP("EHP", new JLabel("--")),
-    EHB("EHB", new JLabel("--")),
-    EXP("Exp", new JLabel("--")),
-    LAST_UPDATED("Last updated", new JLabel("Last updated --"));
+    BUILD("Build", "../build.png", "--"),
+    COUNTRY("Country", "../flags_square/default.png", "--"),
+    TTM("TTM", "../ttm.png", "--"),
+    EHP("EHP", "../ehp.png", "--"),
+    EHB("EHB", "../bosses/ehb.png", "--"),
+    EXP("Exp", "../overall.png", "--"),
+    LAST_UPDATED("Last updated", "", "Last updated --");
 
     private final String rawString;
-    private final JLabel label;
+    private final String iconPath;
```

Comment:
You can load the image here instead of just saving the path and loading it later